index.js

```diff
@@ -0,0 +1,26 @@
+import { formButton, bookList, Book } from './modules/bookClass.js';
+
+import { sections, navList } from './modules/navItems.js';
+
+import * as displayTime from './modules/luxon.js';
+
+const myBook = new Book();
+
+formButton.addEventListener('click', (e) => myBook.handleFormSubmit(e));
+bookList.addEventListener('click', (e) => myBook.handleRemoveButtonClick(e));
+myBook.addBookToList();
+
+for (let i = 0; i < navList.length; i += 1) {
+  navList[i].addEventListener('click', () => {
+    sections[i].classList.add('display');
+    navList[i].classList.add('active');
+    for (let j = 0; j < navList.length; j += 1) {
+      if (i !== j) {
+        sections[j].classList.remove('display');
+        sections[j].classList.add('hidden');
+        navList[j].classList.remove('active');
+      }
+    }
+  });
+}
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md#:~:text=the%20correct%20way.-,Display%20the%20current%20date%20below%20the%20navigation%20bar.,-For%20this%20requirement), Please display the current date below the navigation bar.

| Given wireframe  | Your app |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/235576022-045f703e-4a67-45e3-bd5f-18898442d670.png) | ![image](https://user-images.githubusercontent.com/68796232/235576015-e1bb2bab-6522-46f0-8fcf-08cf74f300d2.png) |